teuthology/task/buildpackages/common.sh

```diff
@@ -41,9 +41,6 @@ function get_ceph() {
 
     test -d ceph || git clone ${git_ceph_url} ceph
     cd ceph
-    if test -d src ; then # so we don't try to fetch when using a fixture
-       git fetch --tags http://github.com/ceph/ceph
-    fi
```

Comment:
IMO http://github.com/ceph/ceph is authoritative when it comes to tags and cloned repository with conflicting tags must be fixed. If not chaos is guaranteed ;-)